Automattic/jetpack
# Blocks: update remaining child blocks to API version v3

Fixes #40678

## Proposed changes:

It seems that we had forgotten to update a few of our child blocks in #34120.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

> [!NOTE] 
> 1. You'll need to test all the blocks touched by this PR on a site running Gutenberg (can be a simple site for example), and not using Gutenberg (can be a fresh JN site).
> 2.  To test the recipe block on non-simple sites, you will need to add a filter to your site on self-hosted (`add_filter( 'jetpack_blocks_variation', function () { return 'beta'; } );`) or use the option in Settings > Jetpack constants on JN. On simple sites, you will get the Beta blocks if you are proxied.

1. Before you switch to this branch, create posts with each one of the touched blocks.
2. Then switch to this branch.
3. Edit each one of the posts and ensure that there are no errors in the console, that the blocks can still be edited, and that they look good on the frontend (still use the right CSS classes).
